website/docs/r/application_gateway.html.markdown

```diff
@@ -281,6 +283,8 @@ A `probe` block support the following:
 
 * `unhealthy_threshold` - (Required) The Unhealthy Threshold for this Probe, which indicates the amount of retries which should be attempted before a node is deemed unhealthy. Possible values are from 1 - 20 seconds.
 
+* `pick_host_name_from_backend_http_settings` - (Optional) Whether the host header should be picked from the backend http settings.

```

Comment:
Could we show the default value to optional attributes?
```suggestion
* `pick_host_name_from_backend_http_settings` - (Optional) Whether the host header should be picked from the backend http settings. Defaults to `false`.
```

Reply:
done